teambuilder/web/models.py

```diff
@@ -44,6 +44,10 @@ class Team(models.Model):
     created_by = models.ForeignKey(Participant)
     description = models.CharField(max_length=200, blank=True)
 
+    @staticmethod
+    def team_count():
```

Comment:
Though not big an issue, better rename it to `def count():`, just for convention. It would be less awesome to write `models.Team.team_count()` instead of `models.Team.count()`.